src/templates/cda/ConsultationNote.hbs

```diff
@@ -20,6 +20,8 @@
         {{>Sections/Medication.hbs}},
         
         {{>Sections/Procedures.hbs}},
+
```

Comment:
Not sure which template to add this to so I added it to all of them.

Reply:
you only need to modify ccd.hbs